compiler/pipes/check-color.h

```diff
@@ -0,0 +1,373 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2020 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#pragma once
+
+#include <utility>
+
+#include "common/algorithms/contains.h"
+#include "common/termformat/termformat.h"
+#include "compiler/compiler-core.h"
+#include "compiler/function-colors.h"
+#include "compiler/function-pass.h"
+
+namespace fp = function_palette;
```

Comment:
`fp` is not an obvious shorthand for the `function_palette` in my opinion.

Reply:
Yep, removed.